intranet/static/js/eighth/admin.js

```diff
@@ -146,16 +147,16 @@ $(function() {
     }
 
     $("#id_restricted").click(updateRestrictedFormFields);
-    if($("#id_restricted").length > 0) {
+
+    if ($("#id_restricted").length > 0) {
         updateRestrictedFormFields();
-    }
+    };
 
     $("#only-show-overbooked").click(function() {
         $("tr.underbooked").toggle();
-    })
+    });
 
     $("#hide-administrative").click(function() {
         $("tr.administrative").toggle();
-    })
-
-});
+    });
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2746691962/issues/source?bid=3176160&fileBranchId=3355598#l162)